recipes/clang_win-64/activate-msvc-headers.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+MSVC_HEADERS_VERSION=14.11.25547
+
+[ -z "${CI}" ] || export CONDA_BUILD_WINSDK=/tmp/cf-ci-winsdk
+
+if [[ -z "${CONDA_BUILD_WINSDK}" ]]; then
+    echo "CONDA_BUILD_WINSDK" is not set.
+    return 0
+fi
+
+echo "By setting CONDA_BUILD_WINSDK, you are agreeing to the terms and conditions of the MSVC headers"
+
+MSVC_HEADERS_DIR=${CONDA_BUILD_WINSDK}/msvc-${MSVC_HEADERS_VERSION}
+
+if [[ ! -d "${MSVC_HEADERS_DIR}" ]]; then
+  mkdir -p "${MSVC_HEADERS_DIR}"
+  pushd "${MSVC_HEADERS_DIR}"
+    mkdir -p tmp
+    pushd tmp
+      curl -L -O https://www.nuget.org/api/v2/package/VisualCppTools.Community.VS2017Layout/${MSVC_HEADERS_VERSION}
+      unzip -o ${MSVC_HEADERS_VERSION} > /dev/null
+      #find lib -type d -name "x86" -delete || true
```

Comment:
What about those commented deletes? Will they be fixed or should they be removed?

Reply:
I was wondering whether we should keep non x64 libs around or not and decided to keep them. I kept the comment here as a reminder that we might want to do it later.